addons/website/static/src/scss/website.edit_mode.scss

```diff
@@ -81,14 +81,34 @@ $-editor-messages-margin-x: 2%;
     }
 
     &.oe_structure.oe_empty, &[data-oe-type=html], .oe_structure.oe_empty {
-        &#wrap:empty, &#wrap > .oe_drop_zone.oe_insert:not(.oe_vertical):only-child {
+        &#wrap:empty, &#wrap > .oe_drop_zone.oe_insert:not(.oe_vertical) {
             @extend %o-editor-messages;
-            padding: 112px 0px;
+
+            &:only-child {
+                padding: 112px 0px;
+            }
```

Comment:
You now apply `&:only-child` to `#wrap:empty` which was not the case before, intended?

Reply:
> if its empty it can't have a only child

That not what `:only-child` means, it checks if you are the only-child yourself. So this would break a layout where `#wrap has a neighbor :wink: